lib/GridItem.jsx

```diff
@@ -676,6 +656,26 @@ export default class GridItem extends React.Component<Props, State> {
       });
     }
 
+    const { x, y, i, maxH, minH } = this.props;
```

Comment:
I had to move this part _after_ the updated size is computed, and used the updated size to compute the new w and h coordinates. It seems more logical like that but I honestly don't know the library enough to make sure this change is OK... 

Thoughts @STRML  ?

Reply:
Yes this is a good change - thanks!